sites/fast-website/src/public/index.html

```diff
@@ -3,12 +3,114 @@
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
     <fast-design-system-provider use-defaults>
-      <fast-button>Hello, World!</fast-button>
-      <gradient-picker>stub</gradient-picker>
+      <div class="wrapper">
+        <!--header-->
+        <header class="header">
+          <div>
+              <button>Logo</button>
+          </div>
+          <div class="header-link-group">
```

Comment:
A `<nav>` element would be more semantically useful here

Reply:
I can definitely update to nav when I start working on that component.

Reply:
I agree. will update when I start on the component.